hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieActiveTimeline.java

```diff
@@ -64,6 +64,8 @@ public interface HoodieActiveTimeline extends HoodieTimeline {
 
   HoodieInstant createRequestedCommitWithReplaceMetadata(String instantTime, String actionType);
 
+  String createCompletionTime();
```

Comment:
add doc for this interface, do we really need this interface? I see the implementation are all the same.